src/components/Card/Card.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  addCardNumberDashes,
+  addSeparator,
+  replaceCardNumberToO,
+} from '@/utils/formatter';
+import { CardContainer } from './Card.style';
+import { CardField } from '@/types';
+import { Colors, colors } from '@/styles/colors';
+
+type CardProps = {
+  size: 'small' | 'big';
+  cardColor: Colors;
+  cardName?: string;
+} & Omit<CardField, 'cardPassword' | 'cvc'>;
+
+const Card = ({
+  size,
+  cardNumber,
+  expirationMonth,
+  expirationYear,
+  cardName,
+  cardColor,
+  ownerName,
+}: CardProps) => {
```

Comment:
CardProps를 보시면 스타일에 관한 prop과 card도메인에 관한 prop이 있어요.
여기서 card쪽 prop을 보면 렌더하는데 필요한 데이터가 많은데요. 요걸 객체형식으로 묶어서 관리해보시면 어떨까 싶어요. 그러면  관련있는 데이터가 하나의 객체에 모이게되면서 prop부도 깔끔해지는 효과가 있습니다,
또 이 카드 객체는 프로그램 전반에서 반복해서 사용하게 될거예요

```suggestion
type CardProps = {
  size: 'small' | 'big';
  cardColor: Colors;
  card: Card
} 

const Card = ({  size, color,  card }: CardProps) => {
```